tests/integration/pull_compare_test.go

```diff
@@ -32,4 +38,33 @@ func TestPullCompare(t *testing.T) {
 	doc := NewHTMLParser(t, resp.Body)
 	editButtonCount := doc.doc.Find(".diff-file-header-actions a[href*='/_edit/']").Length()
 	assert.Greater(t, editButtonCount, 0, "Expected to find a button to edit a file in the PR diff view but there were none")
+
+	onGiteaRun(t, func(t *testing.T, u *url.URL) {
+		defer tests.PrepareTestEnv(t)()
+
+		session := loginUser(t, "user1")
+		testRepoFork(t, session, "user2", "repo1", "user1", "repo1")
+		testCreateBranch(t, session, "user1", "repo1", "branch/master", "master1", http.StatusSeeOther)
+		testEditFile(t, session, "user1", "repo1", "master1", "README.md", "Hello, World (Edited)\n")
+		resp = testPullCreate(t, session, "user1", "repo1", false, "master", "master1", "This is a pull title")
+
+		// the max value on issue_index.yml for repo_id=1 is 5
+		req = NewRequest(t, "GET", "/user2/repo1/pulls/6/files")
```

Comment:
can we get the url from resp in L49?
Or if fixtures changed, it may become a no-tested test.

Reply:
37353eb